src/components/Movies/Movies.js

```diff
@@ -3,7 +3,7 @@ import MovieCard from '../MovieCard/MovieCard';
 import './Movies.css';
 
 const Movies = ( {movieSet, findSingleMovie} ) => {
-    const movies = movieSet.movies
+    const movies = movieSet
```

Comment:
My only comment on this PR is around this renaming. Why is this line necessary? Why not use the `movieSet` variable?